scripts/geoingest.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import datetime
+import hashlib
+import json
+import os
+import requests
```

Comment:
It is. Did you install geoblacklight-vagrant?
Python 2.7.6 (default, Jun 22 2015, 17:58:13) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import requests
>>>